src/naive/datetime/mod.rs

```diff
@@ -732,6 +741,16 @@ impl NaiveDateTime {
         Some(NaiveDateTime { date, time })
     }
 
+    /// Subtracts given [`FixedOffset`] from the current datetime.
```

Comment:
/// Subtracts given `rhs` from the current datetime.

The docstring will generate a link to the type `FixedOffset` in the function signature.

(If I ever contradicted myself in prior reviews with this advice; sorry, I realized this a few weeks ago that I was unnecessarily "overlinking" to types in docstrings. `cargo doc` will always generate links to the crate types in the function signature).

Reply:
On the other hand mentioning the type matches the documentation of the other methods.